scavenge/x/scavenge/abci.go

```diff
@@ -0,0 +1,14 @@
+package scavenge
```

Comment:
this can be removed

Reply:
Think the definitions of `BeginBlocker` and `EndBlocker` are needed even tho they aren't doing anything